amenity-points.mss

```diff
@@ -2311,6 +2314,10 @@
       [feature = 'landuse_military'] {
         text-fill: darken(@military, 40%);
       }
+      [feature = 'boundary_aboriginal_lands'],
+      [feature = 'boundary_protected_area'] {
```

Comment:
it's defined few lines later in a different (proper) color.

Reply:
You're right, only `boundary_aboriginal_lands` should be used here. Fixed.